Nyalotha/Locales/frFR.lua

```diff
@@ -45,15 +45,15 @@ end
 
 L = BigWigs:NewBossLocale("N'Zoth, the Corruptor", "frFR")
 if L then
-	-- L.realm_switch = "Realm Switched" -- When you leave the Mind of N'zoth
+	L.realm_switch = "Changement de royaume." -- When you leave the Mind of N'zoth
 
-	-- L.custom_on_repeating_paranoia_say = "Repeating Paranoia Say"
-	-- L.custom_on_repeating_paranoia_say_desc = "Spam a say message in chat to be avoided while you have paranoia."
+	L.custom_on_repeating_paranoia_say = "Spamming paranoia dire."
+	L.custom_on_repeating_paranoia_say_desc = "Spam un message en dire pour pour signaler que vous avez la paranoia."
 	-- L.custom_on_repeating_paranoia_say_icon = 315927
 
-	-- L.gateway_yell = "Warning: Chamber of Heart compromised. Hostile forces inbound." -- Yelled by MOTHER to trigger mythic only stage
-	-- L.gateway_open = "Gateway Open!"
+	L.gateway_yell = "Alerte. Des entités hostiles approchent de la chambre du Cœur." -- Yelled by MOTHER to trigger mythic only stage
```

Comment:
This yell needs to match what MOTHER yells in-game during the fight - can you confirm you double checked this?

Reply:
I confirme, this is the exact match. Already tested several times on my own version